src/solver/conv/conv_ocl_dir2Dfwd.cpp

```diff
@@ -30,6 +30,8 @@
 #include <miopen/env.hpp>
 #include <miopen/conv/invokers/gen_x_w_y_pad.hpp>
 
+#define WORKAROUND_MIOPEN_1392 (HIP_PACKAGE_VERSION_FLAT >= 6004000000)
```

Comment:
```suggestion
// Disable this solver due to random GPU memory access faults on gfx11 and gfx12
#define WORKAROUND_LWPMIOPEN_1392 (HIP_PACKAGE_VERSION_FLAT >= 6004000000)
```

See the first comment